nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-client-service/src/main/java/org/apache/nifi/distributed/cache/client/CacheClientRequestHandler.java

```diff
@@ -64,6 +64,18 @@ public void channelReadComplete(ChannelHandlerContext ctx) throws IOException {
         }
     }
 
+    @Override
+    public void channelUnregistered(ChannelHandlerContext ctx) {
```

Comment:
Consider `final` argument here

Reply:
Thanks.  I'm used to this check occurring in the Maven build.  I dug into my IDE settings and found the place to flag this.